week-2/extended-project/banzai/todo-list/main.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+require_relative 'cli'
+
+options = ARGV.join(" ") 
+
+if options == ""
+  puts "Listando"
+  puts show
+  
+elsif options.include?("-d")
```

Comment:
What happen if my task is: "recoger libro-devolver caja". It would detect the "-d" and call the delete method?